officeInsightIndex.js

```diff
@@ -7,6 +7,9 @@ var MEETING_ROOM_AVAILABILITY = '/rest/meetingRoom/availability/FIFTY_ONE';
 var MEETING_ROOM_AVAILABILITY_TIME = '?startTime=';
 var MEETING_ROOM_ASSISTANCE = '/rest/meetingRoom/assistance';
 var MEETING_ROOM_BOOK = '/rest/meetingRoom/book/';
+var GTFO_HOST = 'acheron.herokuapp.com';
```

Comment:
Can you put this in a gitignored config file? I don't care so much that this Heroku instance is public, but we will later want to change it to a company-hosted AWS instance, so we'll want it to be both easily configurable and private.

Something like https://github.com/Nase00/gtfo/blob/master/server/environment.js#L28 but don't worry about validation.